src/generators/css-filter.ts

```diff
@@ -78,21 +78,15 @@ export function cssFilterStyleSheetTemplate(filterValue: string, reverseFilterVa
     });
 
     if (!frameURL) {
-        // If user has the chrome issue the colors should be the other way around as of the rootcolors will affect the whole background color of the page
-        const rootColors = hasChromiumIssue501582() && config.mode === FilterMode.dark ? [0, 0, 0] : [255, 255, 255];
-        const [r, g, b] = applyColorMatrix(rootColors, createFilterMatrix(config));
-        const bgColor = {
-            r: Math.round(r),
-            g: Math.round(g),
-            b: Math.round(b),
-            toString() {
-                return `rgb(${this.r},${this.g},${this.b})`;
-            },
-        };
+        const light = [255, 255, 255];
+        // If browser affected by Chromium Issue 501582, set dark background on html
+        const bgColor = !hasPatchForChromiumIssue501582() && config.mode === FilterMode.dark ?
```

Comment:
Here's how it used to be before the fix https://github.com/darkreader/darkreader/pull/2310/files#diff-b3bf9845410975de4e9089094d063eedac8f2a83d9019e4d38723aeeb670a34eL66

So we had a bit false assumption a year ago. Strange that nobody noticed during the whole year.

Reply:
@Gusted 
How does it look in the legacy Chromium build? I'm running this patch in Chromium 90 and it seems to have fixed the issue for me. My interpretation of the logic was that it meant to "manually" set a dark `background` on `html` in the affected versions, or light otherwise, relying on the injected CSS to do the rest.

For what it's worth, I've been seeing the gray background on many pages for quite a while and I think I remember finding at least one open issue about it. I'm not sure how many people use Filter vs Dynamic mode, so hard to say what the impact would have been.

Reply:
> For what it's worth, I've been seeing the gray background on many pages for quite a while and I think I remember finding at least one open issue about it. I'm not sure how many people use Filter vs Dynamic mode, so hard to say what the impact would have been.

A lot people I come across that uses Filter are very frustrated with the performance of Dynamic and don't care about all these fancy colors being darkened etc and just want some dark background with light color with some good options to change brightness etc.



> How does it look in the legacy Chromium build?

Looking good with the latest commits 
![image](https://user-images.githubusercontent.com/25481501/120550862-e9f46580-c3e4-11eb-88ff-6b7fe8e486e9.png)




>  My interpretation of the logic was that it meant to "manually" set a dark `background` on `html` in the affected versions, or light otherwise, relying on the injected CSS to do the rest.

Like Alexander said we had a good false assumption about this whole bug. 

So just everyone on the same page:

We have this example HTML as demo

```HTML
<html style="filter: filter(100%) hue-rotate(180deg)">
<body style="background-color: red">
<p> here some text lalala</p>
</body>
</html>
```
Let's take this legacy version of Chromium:
![image](https://user-images.githubusercontent.com/25481501/120551569-b534de00-c3e5-11eb-806d-b1d6324f294c.png)
You see, the body isn't being effected and the actually red color. 
Wow the bug! Only HTML is being affected by this filter and not it's siblings.

New version of Chromium:
![image](https://user-images.githubusercontent.com/25481501/120551791-f200d500-c3e5-11eb-87c0-6d373084f523.png)
You see, now the red is being affected by the filter! The intended behavior.

As at first it wasn't being affected we had a simple black background-color, because it wouldn't be affecting anything otherwise. However in the new version we had to set a white background-color, because it now WAS affecting other elements and thus causing it to be a white background-color. 

Yeah we are relying on other CSS, but because of mixed behavior across versions we are setting this background-color on newer versions to white-ish.

Reply:
Yeah, I find myself in that camp too—the improved performance is worth it. Though, other than this inverted background issue, I haven't really had much trouble.

Is this working as intended or should we make some more changes?